great_expectations/core/util.py

```diff
@@ -162,7 +181,10 @@ def determine_progress_bar_method_by_environment() -> Callable:
     return tqdm
 
 
-def convert_to_json_serializable(data) -> dict:  # noqa: C901 - complexity 28
+JSONValues: TypeAlias = Union[dict, list, str, int, float, bool, None]
```

Comment:
We might want to consider something like this...
https://github.com/python/typing/issues/182#issuecomment-1320974824